Don't try to re-open log files. This avoids a failure after --daemon
causes a chdir to /.

diff --git a/CumulusService/sources/main.cpp b/CumulusService/sources/main.cpp
index 891621f..39ea4eb 100644
--- a/CumulusService/sources/main.cpp
+++ b/CumulusService/sources/main.cpp
@@ -132,7 +132,6 @@ protected:
 	void dumpHandler(const char* data,int size) {
 		cout.write(data,size);
 		_logStream.write(data,size);
-		manageLogFile();
 	}
 
 	void logHandler(Thread::TID threadId,const std::string& threadName,Priority priority,const char *filePath,long line, const char *text) {
@@ -140,23 +139,6 @@ protected:
 		_logStream << DateTimeFormatter::format(LocalDateTime(),"%d/%m %H:%M:%S.%c  ")
 				<< g_logPriorities[priority-1] << '\t' << threadName << '(' << threadId << ")\t"
 				<< Path(filePath).getFileName() << '[' << line << "]  " << text << std::endl;
-		manageLogFile();
-	}
-
-	void manageLogFile() {
-		if(_logFile.getSize()>LOG_SIZE) {
-			_logStream.close();
-			int num = 10;
-			File file(LOG_FILE(10));
-			if(file.exists())
-				file.remove();
-			while(--num>=0) {
-				file = LOG_FILE_VAR(num);
-				if(file.exists())
-					file.renameTo(LOG_FILE_VAR(num+1));
-			}
-			_logStream.open(LOG_FILE(0),ios::in | ios::ate);
-		}	
 	}
 
 	bool onConnection(Client& client) {
